src/renderer/components/+preferences/preferences.tsx

```diff
@@ -70,6 +82,21 @@ export class Preferences extends React.Component {
     }));
   }
 
+  minimapOptions: SelectOption<Boolean>[] = [
```

Comment:
Please move this options to separate `<EditorSettings>` component to prevent Preferences to grow up.